src/views/home/PlayerHome/components/banner/LabTrainingSlide.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <b-carousel-slide class="slide" :img-src="heroImage">
+    <div class="banner-content">
+      <div class="banner-text">
+        <h2 class="banner-title">
+          {{ title }}
+        </h2>
+        <p>
+          {{  subtitle }}
+        </p>
+        <b-button variant="primary" class="enter-lab" size="lg" :to="training_url">
+          Learn More
+        </b-button>
+      </div>
+    </div>
+  </b-carousel-slide>
+</template>
+
+<script lang="ts">
+  import { Component, Vue, Prop } from 'vue-property-decorator';
+  import DefaultHero from '@/assets/home/hero-lab-default.png';
+  import ToxicRNAHero from '@/assets/home/hero-toxicRNA.jpg';
+  // @ts-ignore
+  // @ts-ignore
+
+  @Component({
+  })
+  export default class LabSlide extends Vue {
+
+    @Prop({ required: true }) readonly title!: string;
+
+    @Prop({ required: true }) readonly subtitle!: string;
+
+    @Prop({ required: true }) readonly training_url!: string;
+
+    get heroImage() {
+      return  ToxicRNAHero || DefaultHero;
```

Comment:
ToxicRNAHero will always be defined here. That said, you parameterized the title, description, link, etc, but not this?

If you want to make this slide parameterized, it might even be better to not make it specific to "lab training" at all (vs just a generic slide with parameters for background, title, subtitle, button text, and button link)

Reply:
Yeah, the slides are not well parameterized or "componentized" in general. I just duplicated the LabSlide and cut out the irrelevant details. I tried to prop the bg image, but ran into issues with the asset loading. So I just copied the EternaCon slide's bg approach.

Reply:
We discussed this in slack, but for the record: Forgot that if you wanted to pass an asset URL as a string (vs an import) in a custom prop, it would need to be added to the transformAssetUrls configuration in our vue config